db/cache/go.mod

```diff
@@ -3,8 +3,12 @@ module github.com/decred/dcrdata/db/cache
 go 1.12
 
 require (
+	github.com/decred/dcrd/chaincfg v1.4.0
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
+	github.com/decred/dcrd/wire v1.2.0
 	github.com/decred/dcrdata/api/types v1.0.7-0.20190416202529-23d1eb95ca1b
+	github.com/decred/dcrdata/blockdata v1.0.1
 	github.com/decred/dcrdata/db/dbtypes v1.0.2-0.20190416202529-23d1eb95ca1b
+	github.com/decred/dcrdata/txhelpers v1.0.2-0.20190416155607-25afe5d5e790
```

Comment:
Would you update the module dependency graph to reflect the added blockdata and txhelpers dependency?

Reply:
Will do.